### ISSUE TITLE
minValue for LineChart

### ISSUE THREAD

**Issue by SparkleFaerieCoder** [Abhinandan-Kushwaha/react-native-gifted-charts#92]
Hi again 👋 

I was wondering if there's a way to provide a minValue to the LineChart? One of the charts I'm trying to implement is a line chart that has no use for values under 100, it's making the chart unreadable so I would rather limit the range shown. 

![Screen Shot 2022-02-23 at 11 34 01 AM](https://user-images.githubusercontent.com/19397130/155394383-97d24a5f-1b9d-4bbb-9093-e82a0d1234ff.png)

**Comment by Abhinandan-Kushwaha** [comment]
Hi Alex🤗
I have a work-around for this.

Let's say your data array is-
```js
const data = [
    {value: 110},
    {value: 130},
    {value: 120},
    {value: 160},
    {value: 190},
  ];
```

If I render the chart using `<LineChart data={data}/>`, the output is-

<img width="312" alt="Screenshot 2022-02-24 at 4 30 42 AM" src="https://user-images.githubusercontent.com/20596944/155423954-4ad44c68-01a1-4622-98a7-7573aa1b13b0.png">

Now, you want the values to start from 100, and they range between 100 to 200.

So you can set the values of the data array to `actual value - 100`
Next you can set custom Y-axis label texts using the `yAxisLabelTexts` prop.

So I modify my data array, and code as shown below-
```js
const data = [
    {value: 110 - 100},
    {value: 130 - 100},
    {value: 120 - 100},
    {value: 160 - 100},
    {value: 190 - 100},
  ];

return (
 <LineChart
    data = {data}
    maxValue = {100}
    yAxisLabelTexts={['100','110','120','130','140','150','160','170','180','190','200']}
  />
```
As you are setting the yAxisLabelTexts, it is required to set the `maxValue` as well. Th maxValue should be 100, because the range of the values is 200-100 = 100.

Now the chart looks like this-

<img width="316" alt="Screenshot 2022-02-24 at 4 32 40 AM" src="https://user-images.githubusercontent.com/20596944/155424192-a329b495-2666-4069-a361-dacd97def28d.png">

Hope it helps!

**Comment by SparkleFaerieCoder** [comment]
Thanks @Abhinandan-Kushwaha, this worked wonderfully. I was headed in this direction, but you helped bridge the gap. I appreciate it 🥇

**Comment by Abhinandan-Kushwaha** [comment]
Hi @SparkleFaerieCoder👋
Now, we can use the prop `yAxisOffset` to achieve this. It's implemented in version 1.2.13🎉

No more work arounds needed!
